### PR TITLE
Tdd2

### PR DESCRIPTION
**Pull request by DaveCasts**
Finished TDD

### CODE REVIEW

File: app/uploaders/picture_uploader.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  # storage :file
+  storage :fog
+
+  # Override the directory where uploaded files will be stored.
+  # This is a sensible default for uploaders that are meant to be mounted:
+  def store_dir
+    "uploads/#{model.class.to_s.underscore}/#{mounted_as}/#{model.id}"
+  end
+
+  # Provide a default URL as a default if there hasn't been a file uploaded:
```
Comment (portatlas) [inline app/uploaders/picture_uploader.rb]: you could get rid of this commented out code block. If the code isn't being used lets not check it in.

File: spec/models/comment_spec.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+
```
Comment (portatlas) [inline spec/models/comment_spec.rb]: Did this assignment also talk about testing models or are they only focusing on testing controllers?

Comment (DaveCasts) [reply spec/models/comment_spec.rb]: It was mostly testing controllers and I believe the last lesson was models but it wasn't part of the site.

File: app/uploaders/picture_uploader.rb (hunk 1 above)
Comment (DaveCasts) [reply app/uploaders/picture_uploader.rb]: Will do when I get a chance.

**Pull request merged**